src/main/java/com/o19s/es/explore/ExplorerQuery.java

```diff
@@ -0,0 +1,243 @@
+            StatisticsHelper idf_stats = new StatisticsHelper();
+            StatisticsHelper ttf_stats = new StatisticsHelper();
+            subWeight.extractTerms(terms);
+
+            for(Term term : terms) {
+                TermContext ctx = TermContext.build(searcher.getTopReaderContext(), term);
+                TermStatistics tStats = searcher.termStatistics(term, ctx);
+                df_stats.add(tStats.docFreq());
+                idf_stats.add(sim.idf(tStats.docFreq(), searcher.getIndexReader().numDocs()));
+                ttf_stats.add(tStats.totalTermFreq());
+            }
+
+            /*
+                If no terms are parsed in the query we opt for returning 0
+                instead of throwing an exception that could break various
+                pipelines.
+             */
+            float constantScore;
+
+            if(terms.size() > 0) {
+                switch (type) {
+                    case ("sum_classic_idf"):
+                        constantScore = idf_stats.getSum();
+                        break;
+                    case ("mean_classic_idf"):
+                        constantScore = idf_stats.getMean();
+                        break;
+                    case ("max_classic_idf"):
+                        constantScore = idf_stats.getMax();
+                        break;
+                    case ("min_classic_idf"):
+                        constantScore = idf_stats.getMin();
+                        break;
+                    case ("stddev_classic_idf"):
+                        constantScore = idf_stats.getStdDev();
+                        break;
+                    case "sum_raw_df":
+                        constantScore = df_stats.getSum();
+                        break;
+                    case "min_raw_df":
+                        constantScore = df_stats.getMin();
+                        break;
+                    case "max_raw_df":
+                        constantScore = df_stats.getMax();
+                        break;
+                    case "mean_raw_df":
+                        constantScore = df_stats.getMean();
+                        break;
+                    case "stddev_raw_df":
+                        constantScore = df_stats.getStdDev();
+                        break;
+                    case "sum_raw_ttf":
+                        constantScore = ttf_stats.getSum();
+                        break;
+                    case "min_raw_ttf":
+                        constantScore = ttf_stats.getMin();
+                        break;
+                    case "max_raw_ttf":
+                        constantScore = ttf_stats.getMax();
+                        break;
+                    case "mean_raw_ttf":
+                        constantScore = ttf_stats.getMean();
+                        break;
+                    case "stddev_raw_ttf":
+                        constantScore = ttf_stats.getStdDev();
+                        break;
+
```

Comment:
could you add :
case "unique_terms_count": constantScore = terms.size(); break;

Thanks!